From af58a474526d5c5de636e779f5fbe175c25ecd23 Mon Sep 17 00:00:00 2001
From: Pekka Vuorela <pekka.vuorela@jollamobile.com>
Date: Fri, 20 Apr 2018 16:17:24 +0300
Subject: [PATCH 38/38] Downgrade Glib dependency

Upstream commit 4aab5fd604e7638 bumped this for dnf backend.
Since it's not built here we can force lower version until Glib
get upgraded.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 2f00d82e9..d53ce6f23 100644
--- a/configure.ac
+++ b/configure.ac
@@ -166,7 +166,7 @@ AM_GLIB_GNU_GETTEXT
 dnl ---------------------------------------------------------------------------
 dnl - Library dependencies
 dnl ---------------------------------------------------------------------------
-GLIB_REQUIRED=2.54.0
+GLIB_REQUIRED=2.48.0
 GIO_REQUIRED=2.16.1
 POLKIT_GOBJECT_REQUIRED=0.98
 
-- 
2.13.6

